master/usage/enabling-ipvs.md

```diff
@@ -18,10 +18,10 @@ However, it comes with some limitations.  In IPVS mode:
   {{site.prodname}} will treat traffic to those ports as host traffic instead
   of pod traffic.
 - {{site.prodname}} does not support Kubernetes services that make use of a
-  locally-assigned ExternalIP.  {{site.prodname}} does support ExternalIPs that
-  are implemented via an external load balancer.
+  locally-assigned ExternalIP for kubernetes 1.10. This is due to an kube-proxy issue
```

Comment:
I think "kubernetes" should have an upper-case K.  Maybe add a v before the versions?

Reply:
I would put ExternalIP in backticks so that it's rendered as code.